src/cache-control.interceptor.ts

```diff
@@ -0,0 +1,25 @@
+import {
```

Comment:
Nit: Should we move this to `src/routes/common/inteceptors`?

Reply:
Agree 100%. Changed in [73f6ad7](https://github.com/safe-global/safe-client-gateway/pull/1356/commits/73f6ad71c0fc47c1625ce77b689eda56ab7e7cc9).